README.md

```diff
@@ -1092,7 +1092,6 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Alwaysdata](https://www.alwaysdata.com/) — 100 MB free web hosting with support for MySQL, PostgreSQL, CouchDB, MongoDB, PHP, Python, Ruby, Node.js, Elixir, Java, Deno, custom web servers, access via FTP, WebDAV and SSH; mailbox, mailing list and app installer included.
   * [Awardspace.com](https://www.awardspace.com) — Free web hosting + a free short domain, PHP, MySQL, App Installer, Email Sending & No Ads.
   * [Bubble](https://bubble.io/) — Visual programming to build web and mobile apps without code, free with Bubble branding.
-  * [Deploy Now](https://deploynow.space) — Deploy smarter. Deploy faster. Deploy Now. - Deploy up to 3 web projects from your GitHub repository for free.
```

Comment:
Here is my reason of removing

![image](https://github.com/ripienaar/free-for-dev/assets/68631458/39323fae-353b-4305-a53a-8e4b1f6b0adc)

Notice to the highlight yellow I marked, it means after first month, dev has to pay $4/month. Because of this, I can say it is common trial plan you can find in anywhere. Therefore I request to remove the service out of the list.

Reply:
Disclaim... It is the software fault, I'm pretty sure I marked the highlight correctly :p